src/hotspot/cpu/s390/vtableStubs_s390.cpp

```diff
@@ -195,7 +195,7 @@ VtableStub* VtableStubs::create_itable_stub(int itable_index) {
 
   // Get receiver klass.
   // Must do an explicit check if offset too large or implicit checks are disabled.
-  address npe_addr = __ pc(); // npe == NULL ptr exception
+  address npe_addr = __ pc(); // npe == null ptr exception
```

Comment:
This also reads a bit strange

Reply:
Read this as "npe means null pointer exception" or however you would like to spell it.